File: Java/bee 1004.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int A, B;
        A = sc.nextInt();
        B = sc.nextInt();
        int PROD = A*B;

        System.out.printf("PROD = %d\n", PROD);
    }
}

File: Kotlin/bee 1004.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import java.util.*
import kotlin.math.pow

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val a = br.readLine().toInt()
    val b = br.readLine().toInt()

    println("PROD = ${a*b}")

}
